api/src/main/resources/messages.properties

```diff
@@ -56,6 +56,12 @@ coreapps.task.mergeVisits.mergeSelectedVisits=Merge Selected Visits
 coreapps.task.mergeVisits.success=Visits merged successfully
 coreapps.task.mergeVisits.error=Failed to merge visits
 
+coreapps.task.deletePatient.label=Delete Patient
+coreapps.task.deletePatient.notAllowed=User does not have permission to delete patient
+coreapps.task.deletePatient.deletePatientSuccessful=Patient has been deleted successfully
+coreapps.task.deletePatient.deleteMessageEmpty=Reason cannot be empty
+coreapps.task.deletePatient.deletePatientUnsuccessful=Failed to delete the patient
```

Comment:
@wluyima @djazayeri Are there any further problems? I received an email about your comments but I can't see it on the diff